ts/displayTodos.ts

```diff
@@ -0,0 +1,13 @@
+import { addTodo } from "./addTodo.js";
+import { tasks } from "./todoapp.js";
+import { isTypeTodo } from "./setSingleTodoDeletionListener.js";
+
+function displayTodos() {
+  if (isTypeTodo(tasks)) {
```

Comment:
Why you check this `is it todo` ? You tasks must always be of todo type, not need to check here, problem is in your `tasks`